### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by mtcazzolato**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by mtcazzolato — COMMENTED** [review]
My comments

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (mtcazzolato) [inline index.html:50]: remove icons, please

Comment (mileratests) [reply index.html:50]: remova os icones por favor

**Review by mileratests — CHANGES_REQUESTED** [review]
Remova emoticons do codigo.

**Pull request merged**